src/components/search/search-box.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Downshift from 'downshift';
```

Comment:
💡Downshift makes our search input and and results navigable by keyboard!

Reply:
ooooooo!